modules/gatsby/website-template/ocular-config-template.js

```diff
@@ -34,6 +42,7 @@ module.exports = {
   PROJECT_DESC: '${init.desc ||
     'A documentation website for Ocular, made with Ocular'}',
   WEBSITE_PATH: '${init.path || '/website/'}',
```

Comment:
Is WEBSITE_PATH deprecated? I don't see it in the documentation.

Reply:
> Is WEBSITE_PATH deprecated? I don't see it in the documentation.

I think it is no longer used. We could remove it and issue a warning when we first load ocular-config.

We should probably have an object with default values for all supported keys and emit warnings if ocular-config contains any unknown keys. That could save users a lot of time. I will open an issue on that.